src/advanced_inv.cpp

```diff
@@ -2177,6 +2177,11 @@ bool advanced_inventory::query_charges( aim_location destarea, const advanced_in
         redraw = true;
         return false;
     }
+    if( it.typeId() == "partial_construction" ) {
+        popup( _( "You can't pick up an unfinished constrution" ) );
```

Comment:
Hmm, we really shouldn't need the special casing here and elsewhere.  Ideally it should be impossible for the player to ever see this item to select it for picking up.